src/stcal/jump/jump.py

```diff
@@ -1,21 +1,19 @@
-import time
 import logging
-import warnings
+import multiprocessing
+import time
 
 import numpy as np
-from . import twopoint_difference as twopt
-from . import constants
 
-import multiprocessing
+from . import constants
+from . import twopoint_difference as twopt
 
+ELLIPSE_PACKAGE = None
 try:
     import cv2 as cv
 
-    OPENCV_INSTALLED = True
-except ImportError:
-    OPENCV_INSTALLED = False
-    warnings.warn('Could not import `opencv-python`; '
-                  'certain snowball detection and usage of ellipses will be inoperable')
+    ELLIPSE_PACKAGE = 'opencv-python'
+except (ImportError, ModuleNotFoundError):
+    ELLIPSE_PACKAGE_WARNING = '`opencv-python` must be installed (`pip install stcal[opencv]`) in order to use ellipses'
```

Comment:
this is the message that will accompanies the `ModuleNotFoundError` if `opencv` functionality is used without it being installed